console/services/plugin/app_plugin.py

```diff
@@ -582,6 +582,9 @@ def build_plugin_data_4marketsvc(self, tenant, service, plugin):
             return
         plugin_id = p[0].plugin_id
         plugin_version = plugin_version_service.get_newest_plugin_version(tenant.tenant_id, plugin_id)
+        if not plugin_version:
```

Comment:
Logging only, no other logic?

Reply:
Yes, here is to avoid grammatical errors, refer to the logic above, ignore this plugin